### ISSUE TITLE
Invalid PDF data: missing %PDF header

### ISSUE THREAD

**Issue by LiThaM** [smalot/pdfparser#497]
Hi, 
I change to word to PDF and receive  error.

"Invalid PDF data: missing %PDF header."

PDF open normal not a problem, I attach the pdf document

[2-Prueba_Word.pdf](https://github.com/smalot/pdfparser/files/7715244/2-Prueba_Word.pdf)

**Comment by k00ni** [comment]
Hi @LiThaM, just to make sure I understand. You generate a PDF using MS Word and when opening it with PDFParser you receive the error `Invalid PDF data: missing %PDF header.`?

**Comment by LiThaM** [comment]
Yes, correctly. I also think there must be some bug in the phpword, because
it totally loses the style.

**Comment by k00ni** [comment]
The PDF you mentioned may be added to our test environment. Is it free of charge and without any obligations?

**Comment by LiThaM** [comment]
Yes, that is correct. The pdf is a test that I generate to check why this
error.

**Comment by rubenvanerk** [comment]
I can't replicate this issue. Could you share the code you are using?

**Comment by LiThaM** [comment]
Friends, thanks for everything, the error was in the PDF converter. Solving
that problem, I never jumped again.

**Comment by k00ni** [comment]
Thank you for your feedback. All the best.

**Comment by Rajashekhar-Kategoud** [comment]
Hi Everyone. 
I'm facing below 
**"Exception - Invalid PDF data: missing %PDF header.

[More information about this error](https://docs.moodle.org/400/en/error/moodle/generalexceptionmessage)

×Debug info:
Error code: generalexceptionmessage
×Stack trace:
line 887 of \lib\Pdfparser\vendor\smalot\pdfparser\src\Smalot\PdfParser\RawData\RawDataParser.php: Exception thrown
line 102 of \lib\Pdfparser\vendor\smalot\pdfparser\src\Smalot\PdfParser\Parser.php: call to Smalot\PdfParser\RawData\RawDataParser->parseData()
line 90 of \lib\Pdfparser\vendor\smalot\pdfparser\src\Smalot\PdfParser\Parser.php: call to Smalot\PdfParser\Parser->parseContent()
line 330 of \user\profile\field\file\field.class.php: call to Smalot\PdfParser\Parser->parseFile()
line 714 of \user\profile\lib.php: call to profile_field_file->edit_save_data()
line 270 of \user\editadvanced.php: call to profile_save_data()"**                                                                                                               can anyone help me to fix this? I've added PdfParser library to moodle, there i'm getting the above mentioned error. Outside the moodle, I',m able to parse the pdf without any error.

**Comment by tdwesten** [comment]
I also have this issue with the PDFs I received from the https://ocr.space/ API.

**Comment by svenfulen** [comment]
This is an issue with the format of the PDF file, i.e. some of the expected metadata is missing from the PDFs.  I am currently looking for a work-around.
